### PR TITLE
[gallery] fix(gallery-viewer): horizontal layouts index 

### PR DESCRIPTION
**Pull request by talevy17**

### CODE REVIEW

File: e2e/tests/media.e2e.js [hunk 1]
```diff
@@ -206,9 +212,13 @@ describe('plugins', () => {
         cy.waitForGalleryImagesToLoad();
         cy.eyesCheckWindow(this.test.parent.title + ' - select all items');
         cy.get(`[data-hook=${GALLERY_SETTINGS.DESELECT}]`).click();
-        cy.dragAndDrop(firstImage, anyImage, 1);
         cy.waitForGalleryImagesToLoad();
         cy.eyesCheckWindow(this.test.parent.title + ' - deselect items');
+        // TODO: stabalize reordering tests
```
Comment (shaulgo) [inline e2e/tests/media.e2e.js:217]: is this left here by purpose?

File: e2e/tests/media.e2e.js [hunk 2]
```diff
@@ -189,7 +195,7 @@ describe('plugins', () => {
     context('organize media', () => {
       it('allow to manipulate the media items', function() {
         const firstImage = `[data-hook=${GALLERY_SETTINGS.IMAGE}]:first`;
-        const anyImage = `[data-hook=${GALLERY_SETTINGS.IMAGE}]`;
+        // const anyImage = `[data-hook=${GALLERY_SETTINGS.IMAGE}]`;
```
Comment (shaulgo) [inline e2e/tests/media.e2e.js:198]: is this left here by purpose?

**Review by shaulgo — APPROVED** [review]

File: e2e/tests/media.e2e.js (hunk 1 above)
Comment (talevy17) [reply e2e/tests/media.e2e.js:217]: tl;dr yes.
it appears there was some mixup when we started using applitools - the drag and drop test and the items selection tests were intertwined and it caused some flakiness issues.
ideally we would want to test the drag and drop (reording) capability but it appears unstable (the images in the gallery might get captured blurry and our waitForImageToLoad mechanism doesn't seem to solve it when the focus is on the settings modal.
I'm trying to work it out on a different branch

File: e2e/tests/media.e2e.js (hunk 2 above)
Comment (talevy17) [reply e2e/tests/media.e2e.js:198]: same as the previous comment

**Pull request merged**